src/App/Pages/RecipesPerCategoryList.elm

```diff
@@ -2,19 +2,19 @@ module App.Pages.RecipesPerCategoryList exposing (view)
 
 import App.Model exposing (..)
 import Html exposing (..)
-import RemoteData exposing (RemoteData(..))
+import RemoteData exposing (RemoteData(..), WebData)
 import App.View.Components exposing (viewRemoteData)
```

Comment:
Hehe, well I couldn't find proper places in atom design for those component bits :)